pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -33,13 +33,9 @@ import (
 )
 
 type NodeUsageMap struct {
-	node             *v1.Node
-	usage            api.ResourceThresholds
-	allPods          []*v1.Pod
-	nonRemovablePods []*v1.Pod
-	bePods           []*v1.Pod
-	bPods            []*v1.Pod
-	gPods            []*v1.Pod
```

Comment:
why are we removing the above info from node usage map?

Reply:
Since all three QoS pods are computed per each node by `classifyPods` when they are needed. Not globally for all nodes. Please see invocation of `classifyPods` in the code.

Reply:
To more clarify on the matter. Right now, all be/b/g pods are computed globally for all nodes. However, only over utilized nodes access all the removed fields. So, instead of classifying the pods for all nodes, they are classified only when over utilized nodes are processed. Thus, there's no longer a need to access all the removed fields. Instead, they are computed on-the-fly when they are needed.

Reply:
I agree with this PR. I was just wondering if that statistics (be, burstable, guaranteed) remains part of node usage info (could be stored 0 for underutilized nodes), then it might be useful sometimes to expose that statistics to end users. Dont remember if we are already doing it now.

Reply:
We can have that as part of follow up PR too. I was just bringing it up for discussion.

Reply:
Good point. Yeah. With the code in this PR, we can just call classifyPods even for remaining nodes in case it became configurable for admins/end users.